requirements/crosshair.in

```diff
@@ -0,0 +1,3 @@
+crosshair-tool==0.0.85
+hypothesis-crosshair==0.0.20
+attrs==24.1.0 # match test.in
```

Comment:
Try `-c test.in` for this!  (maybe it also works for the input pins here, if we unpin and add `-c hypothesis-python/[crosshair]`?)

Reply:
`-c hypothesis-python/[crosshair]` would set a lower bound but not an upper bound right? I was intending to defer crosshair breakage in new versions to our weekly dependency bump job with `crosshair.in`, so the exact version match is key

Reply:
the idea is that we pick up the min-version bound from the constraint, and then actually install from the pinned `.txt` version

Reply:
ahh right because this is a `.in`. got it. will try this

Reply:
Looks like pip-compile treats the `-c hypothesis-python/[crosshair]` syntax as a path, rather than an extra:

```
-c ../hypothesis-python/[crosshair]
-c test.in  # match test.in attrs pin

---

pip._internal.exceptions.InstallationError: Could not open requirements file: [Errno 2] No such file or directory: '/Users/tybug/Desktop/Liam/coding/hypothesis/hypothesis-python/[crosshair]'
```